components/director/internal/domain/apptemplate/resolver.go

```diff
@@ -565,6 +578,69 @@ func (r *Resolver) Webhooks(ctx context.Context, obj *graphql.ApplicationTemplat
 	return r.webhookConverter.MultipleToGraphQL(webhooks)
 }
 
+func (r *Resolver) enrichWebhooksWithTenantMappingWebhooks(in graphql.ApplicationTemplateInput) ([]*graphql.WebhookInput, error) {
+	webhooks := make([]*graphql.WebhookInput, 0)
+	for _, w := range in.Webhooks {
+		if w.Version != nil {
+			tenantMappingWebhooks, err := r.getTenantMappingWebhooks(w.Mode.String(), *w.Version)
```

Comment:
Is there a possibility for w.Version being nil? If yes, this may cause nil pointer dereference. Also L592, L596 contain similar logic.